### ISSUE TITLE
Is fuse part of the container?

### ISSUE THREAD

**Issue by psyciknz** [djmaze/resticker#132]
I've trying to run a restic mount from the docker container but getting:
```
bash-5.1# restic mount --no-lock  /mnt/restic/
repository 8f1e59ba opened successfully, password is correct
fusermount: exit status 1
also, the following messages were logged by a library:
2022/04/28 07:33:03 mount helper error: fusermount: fuse device not found, try 'modprobe fuse' first
unable to umount (maybe already umounted or still in use?): exit status 1: fusermount: failed to clone namespace: Operation not permitted
```

I've tried adding manually apk add fuse, but that didn't work either.   Am I missing something, or is this just too hard inside a container?

**Comment by djmaze** [comment]
@psyciknz You _can_ do fuse mounts in a container, but you need to [give the container `SYS_ADMIN` privilege and allow the fuse device](https://stackoverflow.com/a/49021109). You can also define that in your compose file, see `cap_add` and `devices` in the [compose spec](https://docs.docker.com/compose/compose-file/compose-file-v3/).

That said, personally I rather use `restic restore` inside the container.

**Comment by psyciknz** [comment]
Right, yeah I've used the restore first, but it normally takes me a bit to find the file I want, hence finding mounting a bit quicker, as I can just browse.   Not to worry, it means it wasn't something I was doing wrong.

**Comment by schewara** [comment]
@djmaze Thank you for your answer.
As I was also running into the same error, it might be helpful to have a mention of it in the `README` for clarification.
